pcweb/pages/docs/component.py

```diff
@@ -151,10 +152,24 @@ def get_comment(comments: list[str]):
     "Tuple": "blue",
     "None": "gray",
     "Figure": "green",
+    "Literal": "ruby",
 }
 
+count = 0
+
+import hashlib
+
+
+def get_id(s):
+    global count
+    count += 1
+    s = str(count)
+    hash_object = hashlib.sha256(s.encode())
+    hex_dig = hash_object.hexdigest()
+    return "a_" + hex_dig[:8]
```

Comment:
This seems to be messing up the CI for some reason, it's getting set as the theme on the code block, not sure why